connect/connect-c7/utils-c7/src/test/java/io/miragon/miranum/connect/c7/utils/Camunda7BaseVariableValueMapperTest.java

```diff
@@ -0,0 +1,56 @@
+package io.miragon.miranum.connect.c7.utils;
+
+import org.camunda.community.rest.client.dto.VariableValueDto;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class Camunda7BaseVariableValueMapperTest {
+
+    private final Camunda7BaseVariableValueMapper mapper = new Camunda7BaseVariableValueMapper();
+
+    @Test
+    public void testMap_ShouldReturnMappedVariables() {
+        // Given
+        Map<String, Object> variables = new HashMap<>();
```

Comment:
use same Map definition at all places please

Reply:
I would not extract any test data for different modules. Especially if we do not have that many tests. If we have more tests that use that some time in the future we can think about implementing a TestDataSeeder.

Reply:
I was refering to the syntax. Sry for the misleading formulation.